autoplex/fitting/common/utils.py

```diff
@@ -711,36 +703,39 @@ def nequip_fitting(
 
 def m3gnet_fitting(
     db_dir: Path,
-    path_to_default_hyperparameters: Path | str = MLIP_RSS_DEFAULTS_FILE_PATH,
-    device: str = "cuda",
+    path_to_default_hyperparameters: Path | str = MLIP_DEFAULTS_FILE_PATH,
     ref_energy_name: str = "REF_energy",
     ref_force_name: str = "REF_forces",
     ref_virial_name: str = "REF_virial",
+    device: str = "cpu",
     fit_kwargs: dict | None = None,
 ) -> dict:
     """
     Perform the M3GNet potential fitting.
 
+    This function sets up and executes a python script to perform M3GNet fitting.
+    It also calculates training and testing errors after fitting.
+
     Parameters
     ----------
-    db_dir: Path
-        Directory containing the training and testing data files.
-    path_to_default_hyperparameters : str or Path.
-        Path to mlip-rss-defaults.json.
-    device: str
-        Device on which the model will be trained, e.g., 'cuda' or 'cpu'.
-    ref_energy_name : str, optional
+    db_dir: str or Path
+        Path to directory containing the training and testing data files.
+    path_to_default_hyperparameters: str or Path.
+        Path to mlip-defaults.json.
+    ref_energy_name: str, optional
         Reference energy name.
-    ref_force_name : str, optional
+    ref_force_name: str, optional
         Reference force name.
-    ref_virial_name : str, optional
+    ref_virial_name: str, optional
         Reference virial name.
-    fit_kwargs: dict.
-        optional dictionary with parameters for m3gnet fitting with keys same as
-        mlip-rss-defaults.json.
+    device: str
+        Specify device to use "cuda" or "cpu".
+    fit_kwargs: dict
+        Additional keyword arguments for M3GNet fitting with keys same as
+        those in mlip-defaults.json.
 
-    Keyword Arguments
-    -----------------
+    Tunable hyperparameters
```

Comment:
This is not recognized as a valid header in automatic doc generation, can you please revert it back as it was before?

Reply:
Ah, do you mean we need to capitalise each word in the header?

Reply:
Do you mean that we need to capatalise each word in the header? Or only "Keyword Arguments" is accepted?

Reply:
only "Keyword Arguments" is accepted. There are ways to make other Headers to be recognized as well. It should be defined with `napoleon_custom_sections` in this file
https://github.com/autoatml/autoplex/blob/main/docs/conf.py

I do not have time to look into this, so if we want to have this name as header, one needs to check if this works out as intended by building docs locally with required changes in the conf.py file.

Posting here a list of recognized headers https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html#docstring-sections for your reference.

Reply:
Good to know! I have changed back to the previous version.